airbyte-integrations/connectors/source-stripe/metadata.yaml

```diff
@@ -36,6 +36,12 @@ data:
       5.0.0:
         message: Version 5.0.0 introduces fixes for the `CheckoutSessions`, `CheckoutSessionsLineItems` and `Refunds` streams. The cursor field is changed for the `CheckoutSessionsLineItems` and `Refunds` streams. This will prevent data loss during incremental syncs. Also, the `Invoices`, `Subscriptions` and `SubscriptionSchedule` stream schemas have been updated.
         upgradeDeadline: "2023-12-11"
+      5.4.0:
+        message: Version 5.4.0 introduces fixes for the `Refunds` streams. The cursor field is changed from `created` to `updated`.
```

Comment:
I would phrase this as an added feature and not a fix. Like, "`Refunds`, which previously was incremental on the creation date, now tracks updates as well. In order to do that, the cursor field needs to be updates and a full_refresh is recommended (needed?) to get the updates" or something like that

Reply:
Corrected.